### ISSUE TITLE
Handing off utterance after unicode recognition seems to fail

### ISSUE THREAD

**Issue by zersiax** [mltony/nvda-tonys-enhancements#6]
This is a bit of an odd one and I'm not entirely sure where this happens, as it is an issue within  a particular chain of events which  this  addon  is a part of, and I don't really have the logs to debug this any further.
## To reproduce:

- Get NVDA Translate addon.
- Turn  on automatic language switching when supported in Speech Settings.
- Turn on this addon's setting to switch language based on character map.
- - Hit shift+ctrl+nvda+t to turn on NVDA Translate.
- Have NVDA read a bit of text with the  specified character map.

## Observed Behavior:

This addon seems to process the utterance and then not allow Translate to get at it, as NVDA will just read the line with the character map-specified language instead.

## Expected behavior

Given Translate is meant to translate any utterance NVDA makes, it probably should override this behavior somehow, however, I respect that the way NVDA works would make this difficult. This is more a request for your thoughts than an actual bug report.

## Oddities

Some edge cases seem to indicate  that it's not so much this addon as the way it alters the perceived language of an utterance that breaks here, as seemingly, going to a website with an  explicit language tag seems to break this behavior as well.
I'm not an addon developer (yet), so I'm not sure if I should file this here, against NVDA or against the Translate addon, so definitely feel free to mark this as wontfix if this isn't up to you :)

**Comment by mltony** [comment]
Right, in my understanding translate add-on marks speech with language 
tags, which conflicts with Tony's enhancements addon because it's trying 
to do the same thing. Currently in NVDA there is no good way to have two 
add-ons talk to each other, so at this point it is probably fair to say 
that Tony's enhancements is incompatible with translate.
